client/src/app/components/login-form/login-form.component.html

```diff
@@ -8,8 +8,9 @@ <h2 class="login-text-hint">Already have an account?</h2>
     <mat-form-field class="mt-input-field half" appearance="outline" color="accent">
       <input id="login-password" matInput type="password" placeholder="Password" [formControl]="pwd" />
     </mat-form-field>
-    <button id="login-submit-btn" mat-flat-button class="mt-input-field half" [class.btn-opacity-disabled]="logInForm.invalid"
-      [disabled]="logInForm.invalid">LOGIN</button>
+    <button id="login-submit-btn" mat-flat-button class="mt-input-field half"
+      [class.btn-opacity-disabled]="logInForm.invalid"
+      [disabled]="(logInForm.invalid || submitted) && !appError">LOGIN</button>
```

Comment:
on line 13, there is a lot of logic going on in the [disabled] box, we can move this to a function and test it.